src/main/java/cz/sparko/boxitory/service/filesystem/FilesystemDescriptionProvider.java

```diff
@@ -83,9 +97,21 @@ private DescriptionLine parseLine(String line) {
         if (splittedLine.length != 2) {
             return null;
         }
+
+        if (versionAsTimestamp) {
+            splittedLine[1] = Stream.of(
+                    getDateFromTimestamp(splittedLine[0]),
+                    splittedLine[1]
+            ).filter(s -> s != null && !s.isEmpty()).collect(joining("-"));
+        }
+
         return new DescriptionLine(splittedLine[0], splittedLine[1]);
     }
 
+    private String getDateFromTimestamp(String timestamp) {
+        return new Date(Long.valueOf(timestamp)).toInstant().toString();
```

Comment:
Plese you Java8 date time API.
You can create instant from ```Instant.ofSeconds(long)```, then ```toString``` (as you have now) should be enough.

Reply:
old ```java.util.Date``` is wrong by design. https://www.tutorialspoint.com/java8/java8_datetime_api.htm